test/images/volume/rbd/bootstrap.sh

```diff
@@ -41,6 +41,10 @@ sh ./mds.sh
 # Prepare a RBD volume "foo" (only with layering feature, the others may
 # require newer clients).
 # NOTE: we need Ceph kernel modules on the host that runs the client!
+# As the default pool `rbd` might not be created for a non-amd64 ceph deployment,
+# always create the `rbd` pool to make sure the `rbd import` workable.
+ceph osd pool create rbd 8
+rbd pool init rbd
```

Comment:
I'm not comfortable approving this change. please have someone from sig-storage review this.

Reply:
sure, let me find some one from sig-storage.

Reply:
@chendave I would rather name the `pool` with a string with gives clue about the arch or something. Also, imo, its better to check the pool exist before creating one.

Reply:
thanks @humblec for the suggestion! As we don't have many arch specific config so far, so just add a check there to make sure the `rbd` pools is created. WDYT?

Reply:
Thanks. It should be enough for now, accordingly we can modularize in future. :+1: